modules/virt-analyzing-datavolume-conditions-and-events.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-analyzing-datavolumes-using-events-and-conditions.adoc
+
+[id="virt-analyzing-datavolume-conditions-and-events_{context}"]
+= Analyzing DataVolumes using conditions and events
+
+By inspecting the `Type` field and and analyzing associated fields such as
+`Reason` and `Status`, you determine the condition of the DataVolume
+in relation to persistent volume claims (PVCs), and whether or
+not an operation is actively running. You may also receive messages
+that offer specific details about the DataVolume’s status, and how
+it came to be in its current state.
+
+There are many different combinations of `Reason`, `Status`, `Type`, and `Message`
+conditions. Each must be evaluated in its unique context.
+
+Examples of various condition combinations follow.
+
+* `Bound` – If `Type` is `Bound` and `Status` is `True`, then the PVC is `Bound`.
+If the PVC is not bound, the `Status` is `False`. The conditions `Reason` and `Message`
+present details of the current `Status`. In this example, the PVC is `Bound`,
+so `Reason` is `Bound` and `Status` is `True`. The `Message` indicates which
+PVC owns the DataVolume.
++
+`Message`, in the `Events` section, provides further details including how
+long the PVC has bean bound (`Age`) and by what resource (`From`),
+in this case `datavolume-controller`.
++
+----
+Status:
+	Conditions:
+		Last Heart Beat Time:  2020-07-15T03:58:24Z
+		Last Transition Time:  2020-07-15T03:58:24Z
+		Message:               PVC win10-rootdisk Bound
+		Reason:                Bound
+		Status:                True
+		Type:                  Bound
+
+	Events:
+		Type     Reason     Age    From                   Message
+		----     ------     ----   ----                   -------
+		Normal   Bound      24s    datavolume-controller  PVC example-dv Bound
+----
+
+* `Running` – If `Type` is `Running` and `Status` is `True`, an operation is in the
+process of running, such as the ongoing import, in this example.
++
+`Running` will be `False` if the container is not running. If `Ready` is `True`,
+then `Running` will always be `False`, because a DataVolume cannot be ready
+if an import is running.
++
+In the `Events` section, the `Reason` and `Message` contain detailed information
+about why the operation is not running, in addition to the container exit
+message. In this example, the connection failed due to a `404`, listed in the
+`Events` section’s first `Warning`.
++
+----
+Status:
+	 Conditions:
+		 Last Heart Beat Time:  2020-07-15T04:31:39Z
+		 Last Transition Time:  2020-07-15T04:31:39Z
+		 Message:               Import Complete
+		 Reason:                Completed
+		 Status:                False
+		 Type:                  Running
+
```

Comment:
@awels Both of import succeed and fail show as Import Complete?

Reply:
@awels can you comment?

Reply:
This looks like right. the Status of the condition is false, the condition type is Running, so Running is false, which is correct because the import completed.

Reply:
OK, submitted some more rewrites in this version to clarify.